CHANGELOG.md

```diff
@@ -147,6 +206,10 @@
 * Bump coverallsapp/github-action from 2.3.0 to 2.3.1 [#13766](https://github.com/mixxxdj/mixxx/pull/13766)
 * Bump coverallsapp/github-action from 2.3.1 to 2.3.3 [#13793](https://github.com/mixxxdj/mixxx/pull/13793)
 * Bump coverallsapp/github-action from 2.3.3 to 2.3.4 [#13811](https://github.com/mixxxdj/mixxx/pull/13811)
+* Bump actions/upload-artifact from 4.3.3 to 4.6.0 [#14167](https://github.com/mixxxdj/mixxx/pull/14167)
```

Comment:
I wonder if these github actions changes belong in the release notes. These are not about the product, just about the build system.

Reply:
They don't belong in the final release note. I just kept them for completeness. I consider to remove the whole "Misc Refactorings" branch during the release process like we did for 2.5.